### PR TITLE
Initial PR for removing top Explain This button for feedback

### LINKED ISSUE #3250: Remove top bar 'Explain This'
##### Brief description of problem/feature
We have recently added in better training for users on the mission start screens, so it's about time we removed this old training that no one ever clicked on that we haven't updated in years! It will also clean up the interface a bit, possibly making way for new things like showing the speed limit (#3249)! Here's what it looks like right now:

Screenshot from 2023-05-30 12-48-38
Screenshot from 2023-05-30 12-48-53
Screenshot from 2023-05-30 13-00-32

We also have the same text included in the mobile validate page when you click on the info button at the top-right. We don't have comparable training on mobile yet, so we should probably leave it there for now.

### PR DESCRIPTION
**Pull request by justyeethan**
Resolves #3250 

PR for removing the top bar "Explain This" issue. 

Before:
image

After:
image

### CODE REVIEW

**Comment by misaugstad** [comment]
Some notes:
1. This was meant to remove the entire bar, not just the 'Explain this' button!
2. When deleting the JavaScript, make sure to check if any HTML, CSS, or translation text is no longer being used anywhere else and delete that as well.
3. You've included a couple other random changes that shouldn't be a part of this PR

**Comment by justyeethan** [comment]
@misaugstad The new commit reverted the random changes and also removed the entire bar, without changing anything from the mobile view!

**Comment by misaugstad** [comment]
Great! Did we need to keep all of the text for the Validate page, or is there anything from there that we can remove?

**Comment by justyeethan** [comment]
Is there specific text that you are looking at to remove? It looks good to me, and it seems there's no redundant code from the changes I've made and the functions I've deleted.

File: public/javascripts/SVLabel/src/SVLabel/Main.js [hunk 1]
```diff
@@ -446,14 +444,6 @@ function Main (params) {
         svl.ui.statusMessage.title = $("#current-status-title");
         svl.ui.statusMessage.description = $("#current-status-description");
 
-        // OverlayMessage.
-        svl.ui.overlayMessage = {};
-        svl.ui.overlayMessage.holder = $("#overlay-message-holder");
```
Comment (misaugstad) [inline public/javascripts/SVLabel/src/SVLabel/Main.js:451]: Looks like the HTML element associated with this is still in `explore.scala.html`

**Review by misaugstad — CHANGES_REQUESTED** [review]
Okay I didn't exhaustively go through everything, but I noted at least a few examples where translations/code still need to be removed, and one spot where some CSS was removed that shouldn't have been!

Can you go through everything again with a fine-toothed comb?

File: public/javascripts/SVLabel/src/SVLabel/OverlayMessageBox.js [hunk 2]
```diff
@@ -1,62 +0,0 @@
...
-        }
-    };
-
-    /**
-     * Set the message in the overlay box
-     * @param mode
-     * @returns {*}
-     */
-    this.setMessage =function (mode) {
-        var instruction = i18next.t('top-ui.instruction.' + util.camelToKebab(mode));
```
[the remark this answers is not in the document]
Comment (misaugstad) [reply public/javascripts/SVLabel/src/SVLabel/OverlayMessageBox.js:39]: You also missed these, but I am deleting them now as well.

**Review by misaugstad — APPROVED** [review]
Okay I cleaned up a few remaining things so that we could move on to something else!

**Pull request merged**